cmd/clusterctl/cmd/create_cluster.go

```diff
@@ -134,6 +137,7 @@ func init() {
 	createClusterCmd.Flags().StringVarP(&co.VmDriver, "vm-driver", "", "", "Which vm driver to use for minikube")
 	createClusterCmd.Flags().StringVarP(&co.KubeconfigOutput, "kubeconfig-out", "", "kubeconfig", "Where to output the kubeconfig for the provisioned cluster")
 	createClusterCmd.Flags().StringVarP(&co.ExistingClusterKubeconfigPath, "existing-bootstrap-cluster-kubeconfig", "e", "", "Path to an existing cluster's kubeconfig for bootstrapping (intead of using minikube)")
+	createClusterCmd.Flags().BoolVarP(&co.KindBootstrapper, "kind-bootstrapper", "", false, "Uses kind (sigs.k8s.io/kind) as cluster bootstrapper.")
```

Comment:
would this allow a mixture of `kind-bootsrapper` with flags such as `minikube`?
might be a good idea to warn / error for such cases.

Reply:
Yeah agreed, I think the current flags should probably be revisited given that minikube might be phased out. I was going to bring it up at the next meeting and open an issue. We should probably refactor the flags before v1alpha1.